setup.py

```diff
@@ -9,7 +9,7 @@
     author="Sentry",
     author_email="hello@sentry.io",
     install_requires=["flake8>=3.7.0,<3.8.0"],
-    test_requires=["pytest"],
+    tests_requires=["pytest"],
```

Comment:
tests (plural) require (singular) https://setuptools.readthedocs.io/en/latest/setuptools.html

Reply:
Yeah haha, just noticed that. Gonna force push over.